client/app/bundles/course/assessment/submission/components/GetHelpChatPage/SuggestionChips.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react';
+import { Button } from '@mui/material';
+
+import { getSubmissionId } from 'lib/helpers/url-helpers';
+import { useAppDispatch, useAppSelector } from 'lib/hooks/store';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import { generateLiveFeedback } from '../../actions/answers';
+import { sendPromptFromStudent } from '../../reducers/liveFeedbackChats';
+import { getLiveFeedbackChatsForAnswerId } from '../../selectors/liveFeedbackChats';
+import translations from '../../translations';
+
+interface SuggestionChipsProps {
+  answerId: number;
+}
+
+const SuggestionChips: FC<SuggestionChipsProps> = (props) => {
+  const { answerId } = props;
+
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+
+  const submissionId = getSubmissionId();
+
+  const liveFeedbackChatsForAnswer = useAppSelector((state) =>
+    getLiveFeedbackChatsForAnswerId(state, answerId),
+  );
+
+  const suggestions = liveFeedbackChatsForAnswer?.suggestions ?? [];
+
+  const sendHelpRequest = (message: string): void => {
+    dispatch(sendPromptFromStudent({ answerId, message }));
+    dispatch(
+      generateLiveFeedback({
+        submissionId,
+        answerId,
+        noFeedbackMessage: t(translations.liveFeedbackNoneGenerated),
+        errorMessage: t(translations.requestFailure),
+      }),
+    );
+  };
+
+  return (
+    <div className="scrollbar-hidden absolute bottom-full flex ml-1.5 p-0.5 gap-2 w-full overflow-x-auto">
+      {suggestions.map((suggestion) => (
+        <Button
+          key={suggestion.id}
+          className="bg-white text-xl"
```

Comment:
add `shrink-0` to prevent text from overflowing the button on narrow screens.
![image](https://github.com/user-attachments/assets/0f0f38da-0b37-4200-9a7d-d9406640b215)

Reply:
done